ui/apps/pixano/src/routes/styles.css

```diff
@@ -68,3 +68,12 @@
     --ring: hsl(212.7, 26.8%, 83.9);
   }
 }
+
+@layer base {
```

Comment:
question: was it for configuration that you added an empty layer?

Reply:
It was a mistake, thank you for noticing !